docs/api-v0.5/TableAPI.md

```diff
@@ -271,6 +271,13 @@ Callback that is called when a row is double clicked.
 type: `func`
 
 
+### `onRowContextMenu`
```

Comment:
I don't think you should add this to api-v0.5
This is only being added to 0.6+ right?

Reply:
+1 docs are autogenerated, so we don't need to change this file.